src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/DefaultGeneratedDocumentPublisherTest.cs

```diff
@@ -375,7 +374,7 @@ public object GetService(Type serviceType)
                 throw new NotImplementedException();
             }
 
-            public bool TryGetRequest(long id, [NotNullWhen(true)] out string method, [NotNullWhen(true)] out TaskCompletionSource<JToken> pendingTask)
+            public bool TryGetRequest(long id, out string method, out TaskCompletionSource<JToken> pendingTask)
```

Comment:
These attributes are now ambiguous, because IVT sees the NullableAttributes.cs file, and the project targets .NET 5 so the types are in the runtime. Since they're adding no value here, it was easiest to just get rid of them.